grails-app/controllers/au/org/ala/spatial/service/TasksController.groovy

```diff
@@ -155,6 +155,7 @@ class TasksController {
         def errors = tasksService.validateInput(params.name, input, serviceAuthService.isAdmin(params))
 
         if (errors) {
+            response.status = 400
```

Comment:
400 is bad request -  `server cannot or will not process the request due to something that is perceived to be a client error`. How do you know the `error` is not a backend issue and thus should be a `500` error?